python-package/lightgbm/callback.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, best_iteration, best_score):
 
 # Callback environment used by callbacks
 CallbackEnv = collections.namedtuple(
-    "LightGBMCallbackEnv",
+    "CallbackEnv",
```

Comment:
what mypy warning does this address? I'm confused by this change.

Reply:
The declared type `LightGBMCallbackEnv` is not the same name as the variable that holds that definition. That is unnecessary and confusing. Mypy complains about that.

Reply:
I was asking for the specific warning. I ran `mypy --ignore-missing imports python-package/`, and looks like it is this

> python-package/lightgbm/callback.py:28: error: First argument to namedtuple() should be 'CallbackEnv', not 'LightGBMCallbackEnv'